eth-connector-tests/src/connector.rs

```diff
@@ -56,6 +56,7 @@ async fn test_ft_transfer() {
 #[tokio::test]
 async fn test_ft_transfer_user() {
     let contract = TestContract::new().await.unwrap();
+    contract.unpause_all().await;
```

Comment:
Does moving this call to the `TestContract::new` method make sense?

Reply:
fixed: https://github.com/Near-One/aurora-eth-connector/pull/88/commits/9c3373aeeb4b3cbe5a1d2da4e4da254482105e49